docs/browsers.md

```diff
@@ -16,9 +16,3 @@
 | Opera (GX)     | `opera://about/`    |
 | Microsoft Edge | `edge://version/`   |
 | Arc            | `arc://version/`    |
-
-## Common Issues
-
-> [!NOTE]
-> If you are running into any compatibility issues with your browser, please consider opening
-> a pull request and documenting your issue here!
```

Comment:
Deleted since it isn't relevant to the purpose of the file and is common sense (to open an issue if something is broken).

Reply:
It was put there because there aren't any common issues that have been listed so an empty heading would look weirder, I think it's fine to be explicit with our intentions in that document for anyone coming across it and should be kept in.